source/_patterns/02-components/overflow-menu/overflow-menu~individual-icon.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This should open with "oppositeDirection" or have a "flex-end" to work like the default dropdown

![image](https://github.com/db-ui/core/assets/104347736/190f91c2-39ac-447e-8db9-9dc7ba592211)